armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -94,13 +125,12 @@ def _isTopDummyBlockPresent(self):
                 "Top most block will be artificially chopped "
                 "to preserve assembly height".format(self.linked.a)
             )
-            if "detailedAxialExpansion" in self._converterSettings:  # avoid KeyError
-                if self._converterSettings["detailedAxialExpansion"]:
-                    runLog.error(
-                        "Cannot run detailedAxialExpansion without a dummy block"
-                        "at the top of the assembly!"
-                    )
-                    raise RuntimeError
+            if self._detailedAxialExpansion:
+                runLog.error(
+                    "Cannot run detailedAxialExpansion without a dummy block"
+                    "at the top of the assembly!"
+                )
+                raise RuntimeError
```

Comment:
Add the same message you have as the error into the exception so that the error is clearer when it crashes.

Reply:
addressed. 

https://github.com/terrapower/armi/commit/286c294a6bb6f0f91f5b0ea7c230912f62fbcd29